deploy/kubernetes/charts/secret-storage-backend/values.yaml

```
@@ -0,0 +1,70 @@
+# Default values for secret-storage-backend.
+global:
+  containerRegistry:
+    path: ghcr.io/capactio
+    # Overrides the image tag for all Capact components and extensions. Default is the appVersion.
+    overrideTag: "latest"
+
+image:
+  name: secret-storage-backend
+  pullPolicy: IfNotPresent
+
+supportedProviders:
+  - "dotenv"
+
+additionalEnvs: {}
+
+replicaCount: 1
+
+imagePullSecrets: []
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 50051
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
```

I will also define minimal resources, such as:

```suggestion
resources:
  limits:
    cpu: 100m
    memory: 32Mi
  requests:
    cpu: 30m
    memory: 16Mi
```

(I tested the service with such resources and works fine)